internal/datacoord/channel_checker.go

```diff
@@ -123,7 +123,7 @@ func (c *channelStateTimer) startOne(watchState datapb.ChannelWatchState, channe
 			c.notifyTimeoutWatcher(&ackEvent{ackType, channelName, nodeID})
 			return
 		case <-stop:
-			log.Info("stop timer before timeout",
+			log.Warn("stop timer before timeout",
```

Comment:
Seems that it's normal here, so just keep `Info` level.

Reply:
Got it.